script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt

```diff
@@ -675,7 +690,7 @@ I am disappointed in Elesia.
 -- Page 169, Offset 39902.
 -- 大魔術だけで十分だったものを、必要以上に壊すとは。
 %{i}
-Even though her Grand Magecraft sufficed against Arcueid,
+Even though her grand magecraft sufficed against Arcueid,
  she destroyed her far more than was necessary.
 %{/i}
```

Comment:
Ah, I knew there was a problem here.
Roa calls her "Princess".
>Even though her grand magecraft would have sufficed, 
she destroyed the Princess far more than was necessary.